PubnativeLite/PubnativeLite/Ad Cache/HyBidVideoAdCache.h

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <Foundation/Foundation.h>
+#import "HyBidVideoAdCacheItem.h"
+
+@interface HyBidVideoAdCache : NSObject
+
+@property(nonatomic, strong) NSMutableDictionary *videoAdCache;
+
++ (instancetype)sharedInstance;
+- (void)putVideoAdCacheItemToCache:(HyBidVideoAdCacheItem *)videoAdCacheItem withZoneID:(NSString *)zoneID;
```

Comment:
There is a method called `(void)putAdToCache:(HyBidAd *)ad withZoneID:(NSString *)zoneID;` in `HyBidAdCache`. I thought that can be more informative since there is a similar one. (But of course i can rename it to the desired naming convention)